src/component/calculator.js

```diff
@@ -1,48 +1,45 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 /* eslint-disable */
```

Comment:
- [ ] Kindly remove the eslint linter comment. Linters are there to help us write good code, for example, some of the errors are highlighting indentation, white spaces, spaces, etc.